packages/botonic-plugin-hubtype-analytics/src/event-models/ht-event-webview-end.ts

```diff
@@ -1,21 +1,25 @@
-import { EventAction, EventType, EventWebview, RequestData } from '../types'
+import { EventAction, EventType, EventWebviewEnd, RequestData } from '../types'
 import { HtEvent } from './ht-event'
 
-export class HtEventWebview extends HtEvent {
+export class HtEventWebviewEnd extends HtEvent {
   action: EventAction.WebviewStep | EventAction.WebviewEnd
+  flow_thread_id?: string
   webview_thread_id: string
   webview_name: string
-  webview_step_name?: string
+  webview_end_step_name?: string
+  webview_end_step_n?: number
   webview_end_fail_type?: string
   webview_end_fail_message?: string
 
-  constructor(event: EventWebview, requestData: RequestData) {
+  constructor(event: EventWebviewEnd, requestData: RequestData) {
     super(event, requestData)
     this.type = EventType.WebEvent
     this.action = event.action
+    this.flow_thread_id = event.flowThreadId
     this.webview_thread_id = event.webviewThreadId
     this.webview_name = event.webviewName
-    this.webview_step_name = event.webviewStepName
+    this.webview_end_step_name = event.webviewEndStepName
+    this.webview_end_step_n = event.webviewEndStepNumber
```

Comment:
```suggestion
    this.webview_end_step_name = event.webviewStepName
    this.webview_end_step_n = event.webviewStepNumber
```
Suggestion: Since this two parameters represent the same as `webviewStepName` and `webviewStepNumber` from the WebviewStep Event, why don't we use the same name for both to prevent errors and confusions? Also, I think it would make it easier to implement both events